### DISCUSSION TITLE
Healthcheck endpoint

### DISCUSSION THREAD

**Discussion by wuast94** [lobehub/lobehub#2776] [General Question] [answered]
https://github.com/lobehub/lobe-chat/blob/8164f3ce9d0196d27d84bb498abeb29be907ab03/src/server/routers/index.ts#L11

where is this located? would be nice to have a special healthcheck endpoint for monitoring

**Comment by arvinxx** [comment, marked as answer]
`/trpc/edge/healthcheck` , but it's not stable currently

**Reply by wuast94** [reply]
Why IT isnt stable ?

**Reply by arvinxx** [reply]
> Why IT isnt stable ? 

Because we don't offer official api support now, So we won't have a promise that the pathname don't change. It's an internal path for us to make healthcheck, and it also not mentioned in docs.

**Reply by wuast94** [reply]
Ah ok, so i can use it and Report instabilitis If i Encounter some, but there ist No Support 😊 thanks ☺️ 

But anyway IT would be nice to have an offiziell Test endpoint ☺️
